ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/ValidatorsUtil.java

```diff
@@ -74,7 +74,7 @@ public static boolean is_active_validator(Validator validator, UnsignedLong epoc
    */
   public static List<Integer> get_active_validator_indices(
       List<Validator> validators, UnsignedLong epoch) {
-    List<Integer> active_validator_indices = Collections.synchronizedList(new ArrayList<Integer>());
+    List<Integer> active_validator_indices = Collections.synchronizedList(new ArrayList<>());
```

Comment:
Is there a design reason for using an undefined generic?

Reply:
It's an IntelliJ suggestion